ob-deno.el

```diff
@@ -47,7 +47,7 @@
   :safe #'stringp)
 
 (defvar ob-deno-function-wrapper
-  "Deno.stdout.write(new TextEncoder().encode(Deno.inspect((() => {%s})())));"
+  "Deno.stdout.write(new TextEncoder().encode(JSON.stringify(await (async () => {%s})())));"
```

Comment:
Maybe it could be nicer to have the json formatted for the cases where `ob-deno-read` was not sufficient: 
```suggestion
  "Deno.stdout.write(new TextEncoder().encode(JSON.stringify(await (async () => {%s})(), null, 2)));"
```